tests/waku.ts

```diff
@@ -0,0 +1,13 @@
+import { runInRepo } from '../utils.ts'
+import type { RunOptions } from '../types.d.ts'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'dai-shi/waku',
+		branch: 'dev',
```

Comment:
It doesn't look like this branch exists. I suppose you plan to check later, but just in case. FYI, you can run test locally by `pnpm test waku`.